code/__DEFINES/wounds.dm

```diff
@@ -1,43 +1,108 @@
+
 #define WOUND_DAMAGE_EXPONENT	1.4
 
+/// an attack must do this much damage after armor in order to roll for being a wound (incremental pressure damage need not apply)
+#define WOUND_MINIMUM_DAMAGE		5
+/// an attack must do this much damage after armor in order to be eliigible to dismember a suitably mushed bodypart
+#define DISMEMBER_MINIMUM_DAMAGE	10
+/// any damage dealt over this is ignored for damage rolls unless the target has the frail quirk (35^1.4=145)
+#define WOUND_MAX_CONSIDERED_DAMAGE	35
+
+
 #define WOUND_SEVERITY_TRIVIAL	0 // for jokey/meme wounds like stubbed toe, no standard messages/sounds or second winds
 #define WOUND_SEVERITY_MODERATE	1
 #define WOUND_SEVERITY_SEVERE	2
 #define WOUND_SEVERITY_CRITICAL	3
 #define WOUND_SEVERITY_LOSS		4 // theoretical total limb loss, like dismemberment for cuts
 
-#define WOUND_BRUTE	0
-#define WOUND_SHARP	1
-#define WOUND_BURN	2
+/// any brute weapon/attack that doesn't have sharpness. rolls for blunt bone wounds
+#define WOUND_BLUNT		0
+/// any brute weapon/attack with sharpness = SHARP_EDGED. rolls for slash wounds
+#define WOUND_SLASH		1
+/// any brute weapon/attack with sharpness = SHARP_POINTY. rolls for piercing wounds
+#define WOUND_PIERCE	2
+/// any concentrated burn attack (lasers really). rolls for burning wounds
+#define WOUND_BURN		3
 
 // How much determination reagent to add each time someone gains a new wound in [/datum/wound/proc/second_wind()]
 #define WOUND_DETERMINATION_MODERATE	1
 #define WOUND_DETERMINATION_SEVERE		2.5
 #define WOUND_DETERMINATION_CRITICAL	5
+#define WOUND_DETERMINATION_LOSS		7.5
 
+/// the max amount of determination you can have
 #define WOUND_DETERMINATION_MAX			10
 
-// set wound_bonus on an item or attack to this to disable checking wounding for the attack
+/// set wound_bonus on an item or attack to this to disable checking wounding for the attack
 #define CANT_WOUND -100
 
 // list in order of highest severity to lowest
-#define WOUND_LIST_BONE		list(/datum/wound/brute/bone/critical, /datum/wound/brute/bone/severe, /datum/wound/brute/bone/moderate)
-#define WOUND_LIST_CUT		list(/datum/wound/brute/cut/loss, /datum/wound/brute/cut/critical, /datum/wound/brute/cut/severe, /datum/wound/brute/cut/moderate)
-#define WOUND_LIST_BURN		list(/datum/wound/burn/critical, /datum/wound/burn/severe, /datum/wound/burn/moderate)
+#define WOUND_LIST_BLUNT		list(/datum/wound/blunt/critical, /datum/wound/blunt/severe, /datum/wound/blunt/moderate)
+#define WOUND_LIST_SLASH		list(/datum/wound/slash/critical, /datum/wound/slash/severe, /datum/wound/slash/moderate)
+#define WOUND_LIST_PIERCE		list(/datum/wound/pierce/critical, /datum/wound/pierce/severe, /datum/wound/pierce/moderate)
+#define WOUND_LIST_BURN			list(/datum/wound/burn/critical, /datum/wound/burn/severe, /datum/wound/burn/moderate)
```

Comment:
Could you turn these into global lists?
Lists are unique constructs, if you turn them into global lists you'll have just 4 instances of them (do note this means you can't manipulate them without copying, with something like `GLOB.wound_list.blunt.Copy()`, at the risk of changing it for everyone). Else you'll have as many unique lists as you've placed the define.
For usage such as `pick(WOUND_LIST_BLUNT + WOUND_LIST_SLASH)` global lists work fine.